### ISSUE TITLE
Lack of dataset

### ISSUE THREAD

**Issue by eafon0611** [eafon0611/groupnewspaper#1]
We only have data for one month, but to analyze the title and time, maybe we need more. I think we need data for a whole year. Yan, could you please send email to the data holder to ask for more data?

**Comment by YanWU12336** [comment]
Yes ! I've emailed Claire and got one year's data of Dundee's newspaper.

**Comment by s1770406** [comment]
Great! thank you so much.
